### PR TITLE
change font size with command shortcuts

### PR DESCRIPTION
**Pull request by jhaynie**
Change the font size for the window with standard command + or -

### CODE REVIEW

**Comment by montogeek** [comment]
`command+=` is `Cmd` + `+`?

**Comment by rauchg** [comment]
This is brilliant. We should probably memorize the option @jhaynie. Perhaps `localStorage`?

**Comment by johanbrook** [comment]
👍. 

@rauchg Regarding memorizing the option, that should ideally live in some sort of global config somewhere? (out of scope for this PR, but still). Perhaps related to the `hyperterm.json` mentioned in https://github.com/zeit/hyperterm/issues/14.

**Comment by rauchg** [comment]
On further reflection, it'd be cool to register it as menu accelerators:

**Comment by rauchg** [comment]
@johanbrook yes, was thinking exactly that. But in order to make quick progress I'd like to use `localStorage` for now and merge this sooner rather than later.

File: app/hyperterm.js [hunk 1]
```diff
@@ -263,6 +265,32 @@ export default class HyperTerm extends Component {
...
+    const term = this.refs[`term-${uid}`];
+    if (term) {
+      try {
+        const size = term.term.prefs_.get('font-size');
+        term.term.prefs_.set('font-size', size - 1);
+      } catch (e) {
+        alert(e);
+      }
+    }
+  }
```
Comment (sindresorhus) [inline app/hyperterm.js]: This functions are almost identical. I would DRY them up into one that accepted an argument of whether to increment or decrement.

**Comment by montogeek** [comment]
It should also support switch back to the default size

**Comment by jhaynie** [comment]
i'll add the menu accelerators and DRY up the functions. I think we should memoize these in a separate PR to keep this one focused.

**Comment by jhaynie** [comment]
OK, i added reset behavior and added window shortcuts and DRY up code.

**Comment by rauchg** [comment]
@jhaynie this is beyond beautiful and it works flawlessly. Only nit: we need to let the user know about the font change.

Here's my suggestion:
screen shot 2016-07-04 at 6 46 48 pm

**Comment by rauchg** [comment]
Another minor nit: from the perspective of `<HyperTerm>` it'd be more elegant if we simply alter `<Term fontSize={this.state.fontSize} />` in the render loop.

Then we can capture it in `shouldComponentUpdate` and manually perform the transformation on the local term reference and return `false`.

**Comment by rauchg** [comment]
I'm filing issues for these two things.

**Pull request merged**